examples/cnndailymail_text_summarization/readme.md

```diff
@@ -54,31 +54,21 @@ restart shell
     conda create -n pymarlin python=3.8 -y
     conda activate pymarlin
     conda install pytorch cudatoolkit=10.2 -c pytorch -y # make sure cuda version is same as nvidia-smi
-    mkdir PyMarlin
 
-## 2. Transfer code form local machine to VM
-    scp -P $port -r C:\Users\krkusuk\repos\PyMarlin\pymarlin $user@${machine}:/home/$user/PyMarlin/pymarlin
-    scp -P $port -r C:\Users\krkusuk\repos\PyMarlin\setup.py  $user@${machine}:/home/$user/PyMarlin
-    scp -P $port -r C:\Users\krkusuk\repos\PyMarlin\README.md  $user@${machine}:/home/$user/PyMarlin
-    scp -P $port -r C:\Users\krkusuk\repos\PyMarlin\examples\bart $user@${machine}:/home/$user\PyMarlin\bart 
-
-## 3. Install pymarlin and requirements
+## 2. Install pymarlin and requirements
     > ssh $user@$machine -p $port
     $ conda activate pymarlin
-    $ pip install  ./PyMarlin --force-reinstall
-
-    or
-
-    $ export PYTHONPATH=~/PyMarlin/
-
-
+    $ pip install pymarlin
+    $ git clone https://github.com/microsoft/PyMarlin.git
+    $ pip install -r PyMarlin/examples/bart/requirements.txt
+    Make sure to separately install PyTorch with GPU or CPU: https://pytorch.org/get-started/locally/
 
-## 4. Download data
+## 3. Download data
     $ wget https://cdn-datasets.huggingface.co/summarization/cnn_dm_v2.tgz
     tar -xzvf cnn_dm_v2.tgz
-    cd PyMarlin/bart
+    cd PyMarlin/examples/bart
```

Comment:
this has been changed to cnndailymail_text_Summarization now

Reply:
Thanks! Updated the paths